primitives/arithmetic/src/biguint.rs

```diff
@@ -214,7 +214,7 @@ impl BigUint {
 
 				if let Some(v) = u.checked_sub(v) {
```

Comment:
at some point would be better to have this `v` having a different name instead of shadowing the other `v`

Reply:
I see. there is OK

Reply:
update Some(v) to Some(v1)

Reply:
thanks 👍